### DISCUSSION TITLE
Project Research Report: Adding a Security Audit/Penetration Test. Funding requested.

### DISCUSSION THREAD

**Discussion by AdrienAO** [akash-network/community#511] [General]
GM,

I am Adrien, CEO of [AuditOne](https://www.auditone.io). One of our auditor pools recently published a [project research](https://docsend.com/view/4zjpnqtfyi4eg3ra) about Akash. We think Akash is a great project and would like to add a security audit/penetration test. However, the costs would be a bit too high pro bono. 

Would you be willing to fund this?

What are the main advantages?
- Ensuring reliability and security verified by a third party (identifying and addressing vulnerabiltiies)
- User (& Investor confidence)
- Marketing: The report can be published and we have some campaigns around it

For a cost estimation we would assess the code & applications in more detail and provide you with a quotation.

We have the expertise from other audits in the NEAR & Cosmos ecosystems (e.g. with our partner Aurora) and would like to support the Akash Community with it.

Any feedback is much appreciated.

Thanks,
Adrien

**Comment by hewi333** [comment]
Hi @AdrienAO thanks for dropping in the project research, it's always great to see others taking interest in Akash. If you are trying to seek [Community Pool](https://www.mintscan.io/akash/address/community-pool) funds, I would take a look at requesting community pool funds [best practices](https://github.com/orgs/akash-network/discussions/170). It's important to add timelines, costs, etc so that the community can be fully informed. While there is value to third party audits, ultimately it's up to the community to decide how funds are spent.

**Reply by AdrienAO** [reply]
Thanks a lot, we will look into it!
